resultsdbupdater/utils.py

```diff
@@ -330,6 +330,37 @@ def handle_ci_umb(msg):
                 ('category', msg_body.get('category')),
             ) if value is not None
         }
+    elif item_type == 'redhat-module':
+        msg_body_ci = msg_body['ci']
+
+        # The pagure.io/messages spec defines the NSVC delimited with ':', but koji expects:
+        # - NSV delmited with ':'
+        # - C delimited with '.'
+        nsvc = msg_body['artifact']['nsvc'].replace(':', '-', 2).replace(':', '.')
```

Comment:
The mapping from `N:S:V:C` to `N-S-V.C` is a bit more complicated. Stream name can contain a `-` symbol, and MBS replaces it with `_` when importing to Koji. So `mod:my-stream:20181123:deadcafe` should transform to `mod-my_stream-20181123.deadcafe`.

Reply:
ok, so first replace - with _ and then to the same I did :)

Reply:
Name can also contain dashes and those should be preserved. You need to split it and `replace('-', '_')` only in the `S` part.

Reply:
ah that would not work, just the stream name ... ok, more code needed then. Thanks @lubomir